libs/vscode-ui/components/src/lib/field/field.component.html

```diff
@@ -4,22 +4,28 @@
     {{ field.name }}
   </span>
 </div>
-<ng-container [ngSwitch]="field.type">
-  <angular-console-checkbox
-    *ngIf="field.type === 'boolean'"
-    [field]="field"
-    [(value)]="value"
-  ></angular-console-checkbox>
 
-  <angular-console-select
-    *ngSwitchCase="'enum'"
-    [field]="field"
-    [(value)]="value"
-  ></angular-console-select>
+<angular-console-autocomplete
+  *ngIf="field.type === 'array'"
```

Comment:
I think our logic here is that we want to use an angular-console-select if the number of enum values is 10 or less and an autocomplete if 11 or more. We do not currently map to an `array` data type. 

We could definitely augment the server to do so. The server could check if the number of selectable values is 10 or less and conditionally set the type that way. 

What do you think ?

Reply:
Oh I see. I was thinking that there might be some scenarios where we can provide a list of options, but the user could put in a value outside of that list. Is that not the case, or is there a way we could account for that?

Reply:
I don't think there's any need to account for that currently